group_vars/openstack

```diff
@@ -17,17 +17,19 @@ openstack_auth:
   auth_url: "{{ lookup('env', 'OS_AUTH_URL') }}"
 
 cluster_image: centos-7-20190104
+cluster_floating_network_id: NERCvSE
+
 cluster_deploy_user: centos
 cluster_delegate_role: _member_
-cluster_gw_group: gw
+cluster_gw_group: ephemeral
```

Comment:
Why the change in name?

Reply:
Makes more sense than gw?

Reply:
I’m going to be really obtuse and say `ephemeral-gw` makes most sense... :P

Reply:
Or even `deploy-gw` if we want it to really say what it does.